addons/es/verifactu/verifactu.go

```diff
@@ -67,8 +67,6 @@ func normalize(doc any) {
 	switch obj := doc.(type) {
 	case *bill.Invoice:
 		normalizeInvoice(obj)
-	case *tax.Combo:
-		normalizeTaxCombo(obj)
```

Comment:
You need extreme caution when removing normalizations like this. You're effectively making normalization of the tax combos context specific, which is we want to move away from given that there are multiple types of documents.